### PR TITLE
Support null, true and false standalone type

### PR DESCRIPTION
**Pull request by alikhosravidev**
Support new syntax return type in php8.2
Like this :
public function method(): null {}
public function method(): true {}
public function method(): false {}

### CODE REVIEW

**Comment by imanghafoori1** [comment]
The tests are not enough.
you should also test something like this:
`public function exam(): true { }`
`function (): true { }`
`fn (): true => '_'`

File: tests/AbstractMethodsClassTest.php [hunk 1]
```diff
@@ -53,33 +53,33 @@ public function check_is_abstract_method_test()
...
-        $this->assertEquals(true, $methods[2]['is_abstract']);
-        $this->assertEquals(true, $methods[3]['is_abstract']);
-        $this->assertEquals(true, $methods[4]['is_abstract']);
-        $this->assertEquals(true, $methods[5]['is_abstract']);
-        $this->assertEquals(true, $methods[6]['is_abstract']);
-        $this->assertEquals(true, $methods[7]['is_abstract']);
-        $this->assertEquals(true, $methods[8]['is_abstract']);
-        $this->assertEquals(true, $methods[9]['is_abstract']);
-        $this->assertEquals(true, $methods[10]['is_abstract']);
-        $this->assertEquals(true, $methods[11]['is_abstract']);
```
Comment (imanghafoori1) [inline tests/AbstractMethodsClassTest.php:67]: Do not include these changes in this PR.

Comment (alikhosravidev) [reply tests/AbstractMethodsClassTest.php]: OK, I deleted this commit

**Comment by alikhosravidev** [comment]
I added more tests for class method return types.
but i can't write test for function and anonymous like this:
function (): true { }
fn (): true => '_'

Does php_token_analyzer have the ability to analyze these functions?

**Comment by imanghafoori1** [comment]
You have to create new stub files and new test files from scratch and check for PHP 8.2 version in your test and mark the test as skipped if the version is less than 8.2

**Comment by alikhosravidev** [comment]
Done.

**Comment by imanghafoori1** [comment]
You have to mark the test as skipped if the running PHP version is less than 8.2
use `version_compare($version1, $version2)`

**Comment by alikhosravidev** [comment]
image
I did it

**Pull request merged**